boards/arm/dragonclaw/Kconfig.board

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 Google Inc
```

Comment:
should be "Google LLC" ("Copyright (c) 2022 Google LLC")

Reply:
Done